commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -511,7 +511,7 @@ def _preprocess_refund_object(self, refund: Refund, psp: str) -> Refund:
         else:
             refund["created"] = datetime.datetime.utcfromtimestamp(refund["created"])
 
-        refund["status"] = translate_refund_status_to_transaction_status(refund["status"])
+        refund["status"] = translate_refund_status_to_transaction_status(refund["status"].lower())
```

Comment:
This is being converted to a lowercase inside of the [utility](https://github.com/edx/commerce-coordinator/blob/92428563b75f618c9032b23d07ec88d5eda74186/commerce_coordinator/apps/commercetools/utils.py#L212). No need to do it here.